autometa/common/external/hmmer.py

```diff
@@ -79,11 +80,9 @@ def hmmscan(orfs, hmmdb, outfpath, cpus=0, force=False, parallel=True, log=None)
     if parallel:
```

Comment:
What was the difference in time, and how many CPUs did you use? Below @Sidduppal seems to have had different results. If the difference is negligible, I would favor simplifying with just one option.

Reply:
### 1st Run w/o GNU parallel and 4 cpus

```shell
time python -m autometa.common.external.hmmer --cpus 4 --force tests/data/metagenome.orfs.faa autometa/databases/markers/bacteria.single_copy.hmm autometa/databases/markers/bacteria.single_copy.cutoffs tests/data/metagenome.hmmscan.out tests/data/metagenome.markers.tsv

142.50s user 73.71s system 105% cpu 3:25.45 total
```

```shell
time python -m autometa.common.external.hmmer --cpus 4 --force tests/data/metagenome.orfs.faa autometa/databases/markers/bacteria.single_copy.hmm autometa/databases/markers/bacteria.single_copy.cutoffs tests/data/metagenome.hmmscan.out tests/data/metagenome.markers.tsv

143.17s user 75.06s system 104% cpu 3:28.19 total
```

### 2nd Run with GNU parallel and 4 cpus
```shell
time python -m autometa.common.external.hmmer --cpus 4 --parallel --force tests/data/metagenome.orfs.faa autometa/databases/markers/bacteria.single_copy.hmm autometa/databases/markers/bacteria.single_copy.cutoffs tests/data/metagenome.hmmscan.out tests/data/metagenome.markers.tsv

166.16s user 105.19s system 378% cpu 1:11.72 total
```

### 3rd Run with GNU parallel and all cpus available
```shell
time python -m autometa.common.external.hmmer --parallel --force tests/data/metagenome.orfs.faa autometa/databases/markers/bacteria.single_copy.hmm autometa/databases/markers/bacteria.single_copy.cutoffs tests/data/metagenome.hmmscan.out tests/data/metagenome.markers.tsv

230.71s user 148.27s system 703% cpu 53.875 total
```

### 4th Run w/o GNU parallel and explicitly 8 cpus

```shell
time python -m autometa.common.external.hmmer --cpus 8 --force tests/data/metagenome.orfs.faa autometa/databases/markers/bacteria.single_copy.hmm autometa/databases/markers/bacteria.single_copy.cutoffs tests/data/metagenome.hmmscan.out tests/data/metagenome.markers.tsv

156.35s user 110.63s system 118% cpu 3:45.22 total
```

### 5th Run w/o GNU parallel and "0" cpus
```shell
time python -m autometa.common.external.hmmer --force tests/data/metagenome.orfs.faa autometa/databases/markers/bacteria.single_copy.hmm autometa/databases/markers/bacteria.single_copy.cutoffs tests/data/metagenome.hmmscann.out tests/data/metagenome.markers.tsv

123.77s user 63.02s system 99% cpu 3:07.67 total
```

Reply:
I don't know what to make of these results, but my intuition is that it will probably make more difference for very large files. Perhaps we should just use whatever is simplest to implement for now?

Reply:
I think we will notice a bigger difference with larger files as well. I think it is safest to leave both as an option.

Reply:
OK, let's leave both in for now.